### PR TITLE
Choropleth Integration 

### PR DESCRIPTION
**Pull request by meghatron3000**
# Description

This PR adds a sample chloropleth layer onto our current map. The chloropleth currently imports the tract boundaries from the census website and fills the tracts based off of the response rates from test data(next step is to connect with backend for real data).

# Screenshots: 
*Test Tract in California with response rate of .56* 
Screen Shot 2019-10-29 at 10 38 42 PM

*Test Tract in Florida with response rate of .73* 
Screen Shot 2019-10-29 at 10 53 12 PM

### CODE REVIEW

**Comment by chyku** [comment]
its a CHOROPLETH not CHLOROPLETH _please_

File: frontend/src/components/app.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function App() {
   return (
     <Router>
       <div>
-        <Route path="/" component={Sample} />
+        <Route exact path="/" component={Sample} />
         <Route path="/map" component={MapBox} />
```
Comment (chyku) [inline frontend/src/components/app.js]: It'd be great if we could make the main path the path to the map

File: frontend/src/resources/stateLayers.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    { "sourceLayer": "michigan-0zitwe", "sourceURL": "mapbox://meghabyte.d71hz0zq" },
+    { "sourceLayer": "missouri-al6asq", "sourceURL": "mapbox://meghabyte.bsh422ex" },
+    { "sourceLayer": "massachusetts-1p0zje", "sourceURL": "mapbox://meghabyte.63q51vwl" },
+    { "sourceLayer": "mississippi-3nl299", "sourceURL": "mapbox://meghabyte.18k9f9v1" },
+    { "sourceLayer": "minnesota-1r6mxv", "sourceURL": "mapbox://meghabyte.7gxkg2l1" },
+    { "sourceLayer": "maryland-8hvqn0", "sourceURL": "mapbox://meghabyte.8kyq1v9s" },
+    { "sourceLayer": "maine-1pmoki", "sourceURL": "mapbox://meghabyte.82kwc72j" },
+];
+
+export default stateLayer;
```
Comment (chyku) [inline frontend/src/resources/stateLayers.js]: pls run `npm run format` i'll add ci tests for this later lol

**Review by chyku — CHANGES_REQUESTED** [review]
:+1:

**Review by chyku — APPROVED** [review]

**Review by arpanlaha — APPROVED** [review]
🍪

**Review by caiyingshen — APPROVED** [review]
🤓

**Review by coconut750750 — APPROVED** [review]

**Pull request merged**